stella-boleto/src/main/java/br/com/caelum/stella/boleto/transformer/GeradorDeBoleto.java

```diff
@@ -54,14 +54,10 @@
 	 */
 	public GeradorDeBoleto(Collection<Boleto> boletos) {
 		this.boletos = boletos;
-		try {
-			templateJasper = GeradorDeBoleto.class.getResourceAsStream("/br/com/caelum/stella/boleto/templates/boleto-default.jasper");
-			parametros.put(JRParameter.REPORT_LOCALE, new Locale("pt", "BR"));
-			InputStream template_sub = GeradorDeBoleto.class.getResourceAsStream("/br/com/caelum/stella/boleto/templates/boleto-default_instrucoes.jasper");
-			parametros.put("SUB_INSTRUCOES", JRLoader.loadObject(template_sub));
-		} catch (JRException e) {
-			throw new GeracaoBoletoException(e);
-		}
+                templateJasper = GeradorDeBoleto.class.getResourceAsStream("/br/com/caelum/stella/boleto/templates/boleto-sem-pagador-avalista.jasper");
```

Comment:
O template padrão agora vai ser o `boleto-sem-pagador-avalista`? Se for esse o caso, podemos remover os comentários do template `boleto-default_instrucoes`. Se não, temos que pensar numa estratégia de templates melhor

Reply:
Talvez a estratégia de seleção de templates possa ser feito no #214 e reaproveitado aqui, ou feito aqui e reaproveitado lá.

Reply:
Quis simplificar mostrando que é possível apenas um arquivo jasperreports - subreport é mais complicado de manter. Além de atualizar para nova nomenclatura FEBRABAN.

Reply:
Mas eu fiquei com uma dúvida, esse layout vai atender todos os boletos ou só o do Banco do Brasil?

Reply:
Todos

Reply:
Será que esse layout também atende o #214 ?

Reply:
Segui esse documento https://cmsportal.febraban.org.br/Arquivos/documentos/PDF/Nova_plataforma_cobranc%CC%A7a.pdf . Conclui que pode haver esses templates:
1 - boleto-sem-sacador-avalista;
2 - boleto-sem-sacador-avalista (com recibo para pagador);
3 - boleto-com-sacador-avalista;
4 - boleto-com-sacador-avalista (com recibo para pagador);

Optei por fazer o segundo, com recibo para pagador (acredito que a maioria necessita de recibo).
Obs. errei no nome do template atual, coloquei pagador ao invés de sacador.